client/src/test/scala/com.gu.contentapi.client/GuardianContentClientTest.scala

```diff
@@ -12,7 +12,8 @@ import scala.concurrent.ExecutionContext.Implicits.global
 
 class GuardianContentClientTest extends FlatSpec with Matchers with ScalaFutures with OptionValues with BeforeAndAfterAll with Inside {
 
-  private val api = new GuardianContentClient("test")
+  private val apiKey = System.getenv("API_KEY")
```

Comment:
How do you feel about reading configuration properties from a `conf` file? The file can still read the value from the environment variable but it's abstracted away from the tests and can be easy to override locally.

Reply:
hmm, I'd rather not introduce configuration here